app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsScreen.kt

```diff
@@ -46,6 +47,11 @@ fun EventDetailsScreen(
     val context = LocalContext.current
     val snackbarHostState = LocalSnackbarHostState.current
 
+    val dateFromDialog = rememberMaterialDialogState()
+    val timeFromDialog = rememberMaterialDialogState()
+    val dateToDialog = rememberMaterialDialogState()
+    val timeToDialog = rememberMaterialDialogState()
```

Comment:
You could just have 2 of these

1- dateDialog
1- timeDialog

And just reuse those
No need to create 2 timeDialog and 2 dateDialog

Reply:
Yeah, I thought about that but then stopped on this option, as for me in this way it's more readable code. Or will it degrade performance too much?

Reply:
It wont degrade performance
Its just unnecessary haha